### PR TITLE
Draft net notes

### PR DESCRIPTION
**Pull request by dbpokorny**
These are notes related to adding networking support and distributed PR testing to bwaa

### CODE REVIEW

**Comment by botwillacceptanything** [comment]
#### :ballot_box_with_check: Voting procedure reminder:
To cast a vote, post a comment containing `:+1:` (:+1:), or `:-1:` (:-1:).
Remember, you **must :star:star this repo for your vote to count.**

All comments within this discussion are searched for votes, regardless of the time of posting.
You can cast as many votes as you want, but only the last one will be counted.
(You may consider editing your comment instead of adding a new one.)
Comments containing both up- and down-votes are disregarded.
PR authors automatically count as a :+1: vote.

A decision will be made after this PR has been open for **15** minutes (plus/minus **10** percent, to avoid people timing their votes), and at least **8** votes have been made.
A supermajority of **65%** is required for the vote to pass.

*NOTE: the PR will be closed if any new commits are added after:* ed3ba93ec9fd9ed24a971f95d5d6a80fd25d1fef

**Comment by botwillacceptanything** [comment]
:+1:

**Comment by SpenserJ** [comment]
:+1: I'd like to see this fleshed out and some groundwork laid.

**Comment by dbpokorny** [comment]
@SpenserJ word

Here is a formatted version which is easier on the eyes than the diff: https://github.com/dbpokorny/botwillacceptanything/tree/master/projects/r2d2

**Comment by dos1** [comment]
tl;dr so whatever :+1:

**Comment by mappum** [comment]
:+1:

**Comment by pandzieciol** [comment]
:+1:

**Comment by SamsPepper1** [comment]
:+1: 
This would be really exciting to implement. All power to the botnet!

**Comment by dequis** [comment]
:+1:

**Comment by botwillacceptanything** [comment]
#### <a target="_blank" href="http://thecatapi.com/?id=f7"></a>
:+1: The vote passed! This PR will now be merged into master.

----
**Tallies:**
:+1:: 8 (100%) 
:-1:: 0 (0%)
